test/config/newrelic.yml

```diff
@@ -41,6 +41,15 @@ test:
 
   error_collector:
     enabled: true
+    expected_messages:
+      StandardError:
+        - "test error1"
+        - "test error2"
```

Comment:
Yeah, we could do both of these changes, it wouldn't be a difficult change either, since the class we put in the yaml doesn't actually need to exist since we just compare it as a string. The only thing that gives me any pause about doing that is just that both of those types of things seem kind of like tests on the functionality of YAML.load, rather than how we parse it after loading. But that's also not a huge issue, I don't think there is really much downside to adding the change, just something I thought about.

Reply:
Ahh, that's a good point. We don't really need to test YAML.load. I often make the mistake of trying to write tests about the functionality of third-party libraries.